### ISSUE TITLE
We have installed all coins - all setup correctly

### ISSUE THREAD

**Issue by coinlisting** [peatio/peatio#757]

**Comment by cxla** [comment]
Great, I am researching information about Peatio but it seem not many documents support on new version of linux e.g Ubuntu 18.04 TLS, if you have some documents to share that will be very appreciated sir,

**Comment by coinlisting** [comment]
we can install it where you want

**Comment by somidax** [comment]
@coinlisting Nice work and would want to ask if this supports ERC 20 TOKENS AS WELL?

Should  any have good experience on it as well as can deploy, please kindly reach out via pm or email via coinestate@somidax.net

Thanks

**Comment by coinlisting** [comment]
yes all ERC20 works @somidax give me your skype

**Comment by somidax** [comment]
Please send in a brief details via email provided above and we will schedule a meeting via our feedback

Thanks

**Comment by cxla** [comment]
Do you have demo site of using peatio now? thank you

**Comment by coinlisting** [comment]
but i dont understund what you need

**Comment by pkdcryptos** [comment]
Yes, demo site is helpful where we can see deposit and withdraw of few coins. Especially peatio initial code is only good for BTC coins. Checking other coins in working is better rather seeing funds page or just screenshots. I tried to login at your demo site. It did not send activation to registration email. If mailserver is not setup (Because mailgun sends only to registered reciepents with free account), then please provide some working accounts to login and deposit 0.001 coin and withdraw 0.001 coin. That will be super.

**Comment by coinlisting** [comment]
No problem write to our support@listex.io that send to you demo with over 12 coins installed and all’ worked. In few time we add others, or give me your Skype id

**Comment by pkdcryptos** [comment]
Great. I will do that. I have literally gone through each line of the
peatio code except front end coffee/JavaScript. I am a Ruby guy and All
Ruby files I have gone through it. So I can really validate. One thing I
learned is never change backend code mainly matching . Of course coinrpc
code for additional coins is appreciated. Will check and update team.

**Comment by pkdcryptos** [comment]
Looks like, earlier i tried with test site from link @ peatio.io on my mobile, where signin did not work. That link is no more now, that is good. However the dashboard signin is good from the link i got from listex.io (google authentication). So no issues with signup, let me test with few coins : deposit/withdraw and few small orders. Testing with real coins is better, as all coins test nets may not be operational. Looks promising...but need to do real testing.

**Comment by pkdcryptos** [comment]
To benefit others.. let me mention link I tried that did not work.http://test.listex.io. Looks like that is not the right one I shd try.

**Comment by coinlisting** [comment]
exactly

**Comment by pkdcryptos** [comment]
Why mxo2 is approached for some services by listex folks ? Can you please help understand ? I believe listex is the one providing solutions from sites listex/peatio.io. Right ? Did not quite follow the posts #747 and #748   Sorry if it wastes your time, but it helps to understand fully.

**Comment by coinlisting** [comment]
Can you read post, is simple. I don’t know this man. If you need more info write to support@listex.io that I have a installation in progress

**Comment by johnngo** [comment]
Please email me ngongo_jn@hotmail.com if you can install and deploy peatio

**Comment by go-apollyon** [comment]
Give me your Skype id

**Comment by Griccoin** [comment]
i can install and deploy peatio, message me on https://www.fiverr.com/samsoft00 or Skype oyewoleabayomi@gmail.com

**Comment by samsoft00** [comment]
The right gig for Peatio setup and installation => https://www.fiverr.com/s2/af7d90fffb

**Comment by hacksys1337** [comment]
Few questions on this. Hopefully you will be able to help as it looks like those are all implemented and working.

How would I go about adding a Coin from multi-chain block chain. More specifically where are the apis at in peatio that would allow me to implement one. Or interact with multichains rpc for json requests. Ive followed and implemented the modifications for the custom coin to show up in the funds list, trade, admin, etc. 

From https://gist.github.com/brossi/175f60bd1dc4f99f9373

But is there a guide or something for reference for multi-chain implementation.

Any help is appropriated it.

**Comment by ricardoricci** [comment]
Hi Mark 

Do you set up Peatio with severa coins ?
If yes for how many ?
Regards

**Comment by go-apollyon** [comment]
we have installed a lot of coins and a lot in queue. we have done a big exchange https://listex.io
check it

**Comment by ricardoricci** [comment]
And price to set it up ???
With ETH / Erc20 or without ?Multi currency Multi languagePayment gateway included or not ? Do you have any suggestions ?




Envoyé depuis Yahoo Mail pour iPhone


Le jeudi, juin 28, 2018, 3:11 PM, Mark Weir <notifications@github.com> a écrit :


we have installed a lot of coins and a lot in queue. we have done a big exchange https://listex.io
check it

**Comment by ianeinser** [comment]
@go-apollyon please add my skype ian.einser

**Comment by hacksys1337** [comment]
@go-apollyon Can you please assist me We are having issues with ethereum, Peatio is not detecting the transactions, We have our geth server synced to the latest blocks, generating wallets but when sending to a wallet address the transaction doesnt show up on the exchange (This is done on testnet). Any ideas?

**Comment by algobasket** [comment]
@cxla @somidax @johnngo @happypeter 
Complete Peatio Help and Support
http://peatio.algobasket.com
Free Technical Documentation below :
https://github.com/peatio/peatio/files/2457278/PeatioCryptoExchange.pdf
https://www.youtube.com/watch?v=tpieNIMSBuQ
Contact me on Skype : algobasket
Thanks in advance !!!

**Comment by InfraExEric** [comment]
We've released our ERC20 code for Peatio. 

https://github.com/InfraexDev/EtherMan

**Comment by ThusoMathebe** [comment]
Can you please share your code since you have installed all the coins, it would really save me some money. 
Thusomathebe@outlook.com 

Thanks
